homeassistant/components/netatmo/webhook.py

```diff
@@ -39,16 +38,16 @@ async def handle_webhook(hass, webhook_id, request):
     event_type = data.get(ATTR_EVENT_TYPE)
 
     if event_type in EVENT_TYPE_MAP:
-        await async_send_event(hass, event_type, data)
+        send_event(hass, event_type, data)
 
         for event_data in data.get(EVENT_TYPE_MAP[event_type], []):
-            await async_evaluate_event(hass, event_data)
+            evaluate_event(hass, event_data)
 
     else:
-        await async_evaluate_event(hass, data)
+        evaluate_event(hass, data)
 
 
-async def async_evaluate_event(hass, event_data):
+def evaluate_event(hass, event_data):
```

Comment:
Please still name the functions `async_*` if they should run inside the event loop.

If they are not only internal we should also decorate them with `@callback` so they can be hbdmmandled correctly by our core job schedulers.

Reply:
Alright. They should be called by `async_handle_webhook` only.